app/metrics.yaml

```diff
@@ -7278,3 +7278,31 @@ wallpapers:
     data_sensitivity:
       - interaction
     expires: "2023-05-01"
+
+recently_visited_homepage:
+  history_highlight_opened:
+    type: event
+    description: |
+      User has opened any history highlight from homepage
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/23821
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/23909
+    notification_emails:
+      - android-probes@mozilla.com
+    data_sensitivity:
+      - interaction
+    expires: "2023-05-01"
```

Comment:
Unfortunately I've merged the PR to start using version numbers rather than date.  Please update this now to `expires: 114`.  Thanks

Reply:
It was updated

Reply:
Ah I see.  Please update the data review as well.  Thanks

Reply:
Done